kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/KonanConfig.kt

```diff
@@ -113,6 +113,10 @@ class KonanConfig(
         }
         explicit ?: target.needSmallBinary()
     }
+
+    val hotReloadEnabled: Boolean = configuration.get(BinaryOptions.hotReload) == true
+    val hotReloadServerPort: Int = configuration.get(BinaryOptions.hotReloadServerPort)?.toIntOrNull() ?: -1
```

Comment:
If we would like have the K/N application host the server, then it would not be advisable to have this port passed compilation argument, but probably a env variable which can change (as negotiated by the build tooling)